_locales/_untranslated_en.json

```diff
@@ -0,0 +1,8 @@
+{
+  "messaging_disabled_not_in_group": {
+    "message": "Messaging disabled (you are not in this group)"
+  },
+  "message_not_verified": {
+    "message": "The message was sent with non-verified encryption.. See \"Info\" for details."
```

Comment:
Please fix this typo (2 dots).

Reply:
Thanks, i need to fix this in the other pr to not break the history.